crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -780,6 +791,11 @@ def raise_node_upgrade_error(message = "")
         raise ::Crowbar::Error::Upgrade::NodeError.new(message)
       end
 
+      def raise_live_migration_error(message = "")
+        Rails.logger.error(message)
+        raise ::Crowbar::Error::Upgrade::LiveMigrationError.new(message)
```

Comment:
No, I would keep our exception classes the way we have them. Cleanup might make sense, but for every one.